docs/reference/camera.md

```diff
@@ -178,11 +178,17 @@ Moreover an algorithm computing the spherical projection is applied on the resul
 
 So this mode is costly in terms of performance! Reducing the resolution of the cameras and using a `fieldOfView` which minimizes the number of activated cameras helps a lot to improve the performance if needed.
 
+%figure "Spherical camera image"
+
+![spherical_camera_image.png](images/spherical_camera_image.png)
+
 When the camera is spherical, the image returned by the `wb_camera_get_image` function is a 2-dimensional array (s,t) in spherical coordinates.
 
-Let `hFov` be the horizontal field of view, and let `theta` be the angle in radian between the `(0, 0, -z)` relative coordinate and the relative coordinate of the target position along the `xz` plane relative to the camera, then `s=0` corresponds to a `theta` angle of `-hFov/2`, `s=(width-1)/2` corresponds to a `theta` angle of 0, and `s=width-1` corresponds to a `theta` angle of `hFov/2`.
+Let assume a 3D target point is located at coordinates `(X, Y, Z)` relative to the camera origin.
+
+Let `hFov` be the horizontal field of view, and let `theta` be the angle in radian between the `(X, 0, 0)` relative coordinate and the relative coordinate of the target position along the `xy` plane relative to the camera, i.e., `(X, Y, 0)`, then `s=0` corresponds to a `theta` angle of `hFov/2`, `s=(width-1)/2` corresponds to a `theta` angle of 0, and `s=width-1` corresponds to a `theta` angle of `-hFov/2`.
```

Comment:
Yes, it is the angle between the vector from point (0, 0, 0) to point (X, 0, 0) and the vector from point (0, 0, 0) and point (X, Y, 0).

I agree that an angle between coordinates doesn't make sense.
This was already the case before. I will improve it.

I would keep the reference to the plane projection because I think it is useful.